src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -652,6 +652,11 @@ private function registerWorkflowConfiguration(array $config, ContainerBuilder $
             $definitionDefinition->addArgument($transitions);
             $definitionDefinition->addArgument($initialMarking);
             $definitionDefinition->addArgument($metadataStoreDefinition);
+            $definitionDefinition->addTag('workflow.definition', [
+                'name' => $name,
+                'type' => $type,
+                'marking_store' => $workflow['marking_store']['type'] ?? null,
```

Comment:
IMHO, this should be removed. Do you need it?

Reply:
no, I just added the fields that were present in 4.2